### DISCUSSION TITLE
no module named message-ix

### DISCUSSION THREAD

**Discussion by PelinaP** [iiasa/message_ix#606] [Getting started]
This is the first part of my code, 

# load required packages 
import itertools
import pandas as pd

import matplotlib.pyplot as plt
%matplotlib inline
plt.style.use('ggplot')

import ixmp as ix
import message_ix

from message_ix.utils import make_df
from tools import Plots

When i run it, i get the error message 

ModuleNotFoundError                       Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_15228/3332267660.py in <module>
      8 
      9 import ixmp as ix
---> 10 import message_ix
     11 
     12 from message_ix.utils import make_df

ModuleNotFoundError: No module named 'message_ix'

```from message_ix.utils
                         ^
SyntaxError: invalid syntax


#### Versions
version 3.5.0
<details><summary>Output of <tt>message-ix show-versions</tt></summary>
(base) C:\Users\energy>message-ix show-versions
'message-ix' is not recognized as an internal or external command,
operable program or batch file.
```

</details>
[W 12:18:23.864 NotebookApp] 404 GET /api/kernels/8787b665-a3b7-45fd-bc41-78c0841b5a5c/channels?session_id=59b8b4079d44476c840b224a482fcefb (::1): Kernel does not exist: 8787b665-a3b7-45fd-bc41-78c0841b5a5c
[W 12:18:23.901 NotebookApp] 404 GET /api/kernels/8787b665-a3b7-45fd-bc41-78c0841b5a5c/channels?session_id=59b8b4079d44476c840b224a482fcefb (::1) 38.950000ms referer=None
[W 12:18:28.633 NotebookApp] Replacing stale connection: 8787b665-a3b7-45fd-bc41-78c0841b5a5c:59b8b4079d44476c840b224a482fcefb
[I 12:18:33.104 NotebookApp] Kernel started: b1733d86-1625-4556-b1bb-34c50bf2c117, name: python3

**Comment by khaeru** [comment]
I have converted this to a discussion question, because it is not a bug or issue in `message_ix`, but rather user error in installing or using `message_ix` and/or Jupyter.
